project/ws/app/src/lib/routes/events/components/event-list-view/event-list-view.component.ts

```diff
@@ -79,6 +85,13 @@ export class EventListViewComponent implements OnInit, AfterViewInit, OnChanges
     } else {
       this.dataSource.filter = ''
     }
+    this.events.raiseInteractTelemetry(
+      'click',
+      'card-cardContent',
```

Comment:
Card card content doesn't seem right here, create user btn it is right? So better add some relevant subtype that has meaning to what it does